src/readinglistbar.cpp

```diff
@@ -60,13 +74,45 @@ void ReadingListBar::setupList()
     }
 }
 
-void ReadingListBar::on_itemActivated(QListWidgetItem* item)
+void ReadingListBar::onItemClicked(QListWidgetItem* item, Qt::MouseButtons) // Receives single clicks
+{
+    QDateTime thisClickTime = QDateTime::currentDateTime();
+    int doubleClickInterval = QApplication::doubleClickInterval();
+    QTimer::singleShot(doubleClickInterval, [=]() { // Give time for a double click be handled
+        const qint64 timePassed = lastDoubleClickTime.msecsTo(thisClickTime);
+        if (timePassed > doubleClickInterval) {
+            openUrl(item, false);
+        }
+    });
```

Comment:
Now that detection of a double click is performed by Qt, you can replace `lastDoubleClickTime` with simpler logic (e.g. `int clickKind`, set it to 1 in `onItemClicked()` and to 2  in `onItemDoubleClicked()`).

Reply:
Good idea!